apps/dh/api-dh/source/DataHub.WebApi/Controllers/MarketParticipantControllerBase.cs

```diff
@@ -46,7 +46,7 @@ protected async Task<ActionResult> HandleExceptionAsync(Func<Task> func)
             }
             catch (MarketParticipantBadRequestException ex)
             {
-                return StatusCode(ex.StatusCode, ex.JsonError);
+                return StatusCode(ex.StatusCode, ex.JsonError.ToString());
```

Comment:
why the `.ToString()`?

Reply:
I did not get the error message out in my response

Reply:
Jeg graver lidt i det, det her har nemlig virket før